index.html

```diff
@@ -329,9 +328,10 @@
           </pre>
         </div>
         <p>
-          The <a>PaymentRequest</a> constructor MUST act as follows:
+          The <a>PaymentRequest(methodData, details, options)</a> constructor
```

Comment:
Yep, have a patch for ReSpec to start linking constructors. Will land hopefully in a week or so.